app/renderer/actions/Session.js

```diff
@@ -321,6 +321,23 @@ export function newSession (caps, attachSessId = null) {
         }
         https = session.server.testingbot.ssl = true;
         break;
+      case ServerTypes.experitest: {
+        if (!session.server.experitest.url || !session.server.experitest.accessKey) {
+          notification.error({
+            message: 'Error',
```

Comment:
should the `Error` label be translatable?

Reply:
Thanks, you are right!